pkg/controller/windowsmachineconfig/nodeconfig/nodeconfig.go

```diff
@@ -61,43 +83,44 @@ func (nc *nodeConfig) Configure() error {
 }
 
 // configureNetwork configures k8s networking in the node
+// we are assuming that the WindowsVM and node objects are valid
 func (nc *nodeConfig) configureNetwork() error {
-	if nc.GetCredentials() == nil {
-		return fmt.Errorf("nil credentials for VM")
-	}
-
-	if nc.GetCredentials().GetIPAddress() == "" {
-		return fmt.Errorf("empty IP for VM: %v", nc.GetCredentials())
-	}
-
-	node, err := nc.getNode(nc.GetCredentials().GetIPAddress())
-	if err != nil {
-		return errors.Wrap(err, fmt.Sprintf("error getting node object for VM %s",
-			nc.GetCredentials().GetInstanceId()))
-	}
-
 	// Wait until the node object has the hybrid overlay subnet annotation. Otherwise the hybrid-overlay will fail to
 	// start
-	if err = nc.waitForNodeAnnotation(node.GetName(), HybridOverlaySubnet); err != nil {
+	if err := nc.waitForNodeAnnotation(HybridOverlaySubnet); err != nil {
 		return errors.Wrap(err, fmt.Sprintf("error waiting for %s node annotation for %s", HybridOverlaySubnet,
-			node.GetName()))
+			nc.node.GetName()))
 	}
 
 	// NOTE: Investigate if we need to introduce a interface wrt to the VM's networking configuration. This will
 	// become more clear with the outcome of https://issues.redhat.com/browse/WINC-343
 
 	// Configure the hybrid overlay in the Windows VM
-	if err = nc.Windows.ConfigureHybridOverlay(node.GetName()); err != nil {
-		return errors.Wrap(err, fmt.Sprintf("error configuring hybrid overlay for %s", node.GetName()))
+	if err := nc.Windows.ConfigureHybridOverlay(nc.node.GetName()); err != nil {
+		return errors.Wrap(err, fmt.Sprintf("error configuring hybrid overlay for %s", nc.node.GetName()))
 	}
 
 	// Wait until the node object has the hybrid overlay MAC annotation. This is required for the CNI configuration to
 	// start.
-	if err = nc.waitForNodeAnnotation(node.GetName(), HybridOverlayMac); err != nil {
+	if err := nc.waitForNodeAnnotation(HybridOverlayMac); err != nil {
 		return errors.Wrap(err, fmt.Sprintf("error waiting for %s node annotation for %s", HybridOverlayMac,
-			node.GetName()))
+			nc.node.GetName()))
 	}
+	return nil
+}
 
+// applyWorkerLabel applies the worker label to the Windows Node we created.
+func (nc *nodeConfig) applyWorkerLabel() error {
+	if _, found := nc.node.Labels[WorkerLabel]; found {
+		log.V(5).Info("worker label %s already present on node %s", WorkerLabel, nc.node.GetName())
```

Comment:
nit: Why is this V(5)? @mansikulkarni96 I think you were against this?

Reply:
Please fix and follow whatever standard we have

Reply:
yes missed this, it is recommended by logr package we use V(1) if we want them to be debug logs instead of higher levels.